pkgs/test/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: test
-version: 1.16.0-nullsafety.2
+version: 1.16.0-nullsafety.3
```

Comment:
I assume this means that users won't see the help text changes until they're using a null safe version of package:test?

Reply:
That I don't know. @natebosch

Reply:
That is correct, but using a null safe compatible version of `package:test` does not require that the user migrates to null safety.